example/django/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Bugsnag Python: Django example</title>
+</head>
+
+<body>
+  <div>
+    <h1>Bugsnag Python: Django example</h1>
+  </div>
+  <div>
+    <h3>This page is a basic example of how to include bugsnag-python in a django app.
+    </h3>
+  </div>
+
+    <br>
+  <h3>Click below to send errors to your Bugsnag dashboard.</h3>
+  <br>
+  <br>
+    <div id="python">
+        <a href="crash/">
+          <button type="button" id="indexError">UNHANDLED ERROR</button>
+        </a>
+        <br>
+        <a href="/crashcallback">
+          <button type="button" id="indexError">UNHANDLED WITH METADATA</button>
+        </a>
+        <br>
+        <a href="/handled">
+          <button type="button" id="typeError">HANDLED ERROR</button>
+        </a>
+        <br>
+        <a href="/notifywithcontext">
+          <button type="button" id="typeError">MANUAL NOTIFICATION</button>
```

Comment:
`MANUAL NOTIFICATION` doesn't give a great indication of what the button is actually doing.  It would be good to differentiate between that and `HANDLED ERROR`.